src/alire/os_linux/alire-platforms-current__linux.adb

```diff
@@ -20,7 +22,22 @@ package body Alire.Platforms.Current is
    begin
       if Distro_Cached then
          return Cached_Distro;
-      elsif not GNAT.OS_Lib.Is_Regular_File (OS_Identity_File) then
+      end if;
+
+      --  Try to get the distribution ID from an environment variable
+      --  Rationale: Map AlmaLinux, CentOS, Rocky Linux, Oracle Linux
+      --  et al to RHEL.
+
+      begin
+         Cached_Distro := Alire.Platforms.Distributions'Value
+           (Ada.Environment_Variables.Value ("ALIRE_DISTRIBUTION"));
+         Distro_Cached := True;
+         return Cached_Distro;
```

Comment:
In case of success, I'd like to have here a `Trace.Debug` report that the distro is being overridden via environment.